torch/nested/_internal/ops.py

```diff
@@ -29,11 +29,14 @@ def _outer_to_inner_dim(ndim, dim, ragged_dim, canonicalize=False):
     if canonicalize:
         dim = canonicalize_dims(ndim, dim)
 
-    assert dim >= 0 and dim < ndim
+    assert dim >= 0 and dim < ndim  # pyrefly: ignore  # unsupported-operation
 
     # Map dim=0 (AKA batch dim) -> packed dim i.e. outer ragged dim - 1.
     # For other dims, subtract 1 to convert to inner space.
-    return ragged_dim - 1 if dim == 0 else dim - 1
+    return (
+        # pyrefly: ignore  # unsupported-operation
```

Comment:
I noticed sometimes we wrap like this (two ignores wrapping a statement).  Is this related to the line breaks?

Reply:
Nice catch! Removed the extra ones :)